test/abstract_client.js

```diff
@@ -2394,7 +2404,10 @@ module.exports = function (server, config) {
       function check () {
         if (serverPublished && clientCalledBack) {
           client.end()
+          debug('calling done')
           done()
+        } else {
+          debug('serverPublished %s and clientCalledBack %s', serverPublished, clientCalledBack)
         }
```

Comment:
I would not include debug statements in tests. Why were they added?

Reply:
will remove.